pkg/ssh/connection_golang.go

```diff
@@ -324,3 +356,20 @@ func getUDS(uri *url.URL, iden string) (string, error) {
 	}
 	return info.Host.RemoteSocket.Path, nil
 }
+
+// writeKnownHosts writes to the given users' known_hosts file
+func writeKnownHosts(host string, pubKey ssh.PublicKey) (string, error) {
+	hd := homedir.Get()
+	known := filepath.Join(hd, ".ssh", "known_hosts")
+	f, err := os.OpenFile(known, os.O_APPEND|os.O_WRONLY, 0o600)
+	if err != nil {
+		return "", err
+	}
+	defer f.Close()
+	// normalize the line, then add it to known_hosts
```

Comment:
This comment refers to an already removed `Normalized` call.

Reply:
removed, thanks